civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc

```diff
@@ -10,6 +10,20 @@
 function civihr_employee_portal_features_views_default_views() {
   $export = array();
 
+  addLeaveAndAbsenceReportView($export);
```

Comment:
@deb1990 why this functions are added ?

Reply:
> Refatored civihr_employee_portal_features.views_default.inc and moved addition of views to separate functions.

These are added to better manage the views. Previously in the same function, we were adding 4 views, and it was really tough to manage.

Reply:
@deb1990 we should avoid adding/changing feature files directly, adding other functions are not recommended.